### PR TITLE
Remove ACTIVE from default state filters

### PR DESCRIPTION
**Pull request by asieira**
Since ACTIVE is just an alias for including OPEN and IN_PROGRESS, it is redundant to add it to the default state filters.

### CODE REVIEW

File: CLI.md [hunk 1]
```diff
@@ -266,7 +267,7 @@ $ zanshin alert list [OPTIONS] ORGANIZATION_ID
 **Options**:
 
 * `--scan-target-id UUID`: Only list alerts from the specifiedscan targets.
-* `--states [OPEN|ACTIVE|IN_PROGRESS|RISK_ACCEPTED|MITIGATING_CONTROL|FALSE_POSITIVE|CLOSED]`: Only list alerts in the specified states.  [default: OPEN, ACTIVE, IN_PROGRESS, RISK_ACCEPTED, MITIGATING_CONTROL, FALSE_POSITIVE]
+* `--states [OPEN|ACTIVE|IN_PROGRESS|RISK_ACCEPTED|MITIGATING_CONTROL|FALSE_POSITIVE|CLOSED]`: Only list alerts in the specified states.  [default: OPEN, IN_PROGRESS, RISK_ACCEPTED, MITIGATING_CONTROL, FALSE_POSITIVE]
```
Comment (edpin) [inline CLI.md:270]: It still shows ACTIVE here.

File: CLI.md [hunk 2]
```diff
@@ -294,9 +295,16 @@ $ zanshin alert list_following [OPTIONS] ORGANIZATION_ID
 
 **Options**:
 
-* `--following-ids UUID`: Only list alerts from the specifiedscan targets.
-* `--states [OPEN|ACTIVE|IN_PROGRESS|RISK_ACCEPTED|MITIGATING_CONTROL|FALSE_POSITIVE|CLOSED]`: Only list alerts in the specified states.  [default: OPEN, ACTIVE, IN_PROGRESS, RISK_ACCEPTED, MITIGATING_CONTROL, FALSE_POSITIVE]
-* `--severity [CRITICAL|HIGH|MEDIUM|LOW|INFO]`: Only list alerts with thespecified severities  [default: CRITICAL, HIGH, MEDIUM, LOW, INFO]
+* `--following-ids UUID`: Only list alerts from the specified scan targets.
+* `--states [OPEN|ACTIVE|IN_PROGRESS|RISK_ACCEPTED|MITIGATING_CONTROL|FALSE_POSITIVE|CLOSED]`: Only list alerts in the specified states.  [default: OPEN, IN_PROGRESS, RISK_ACCEPTED, MITIGATING_CONTROL, FALSE_POSITIVE]
```
Comment (edpin) [inline CLI.md:299]: Still shows ACTIVE.

File: CLI.md [hunk 3]
```diff
@@ -316,7 +324,7 @@ $ zanshin alert list_grouped [OPTIONS] ORGANIZATION_ID
 **Options**:
 
 * `--scan-target-id UUID`: Only list alerts from the specifiedscan targets.
-* `--state [OPEN|ACTIVE|IN_PROGRESS|RISK_ACCEPTED|MITIGATING_CONTROL|FALSE_POSITIVE|CLOSED]`: Only list alerts in the specified states.  [default: OPEN, ACTIVE, IN_PROGRESS, RISK_ACCEPTED, MITIGATING_CONTROL, FALSE_POSITIVE]
+* `--state [OPEN|ACTIVE|IN_PROGRESS|RISK_ACCEPTED|MITIGATING_CONTROL|FALSE_POSITIVE|CLOSED]`: Only list alerts in the specified states.  [default: OPEN, IN_PROGRESS, RISK_ACCEPTED, MITIGATING_CONTROL, FALSE_POSITIVE]
```
Comment (edpin) [inline CLI.md:327]: Same here.

File: CLI.md [hunk 4]
```diff
@@ -337,7 +345,7 @@ $ zanshin alert list_grouped_following [OPTIONS] ORGANIZATION_ID
 **Options**:
 
 * `--following-ids UUID`: Only list alerts from thespecified scan targets.
-* `--state [OPEN|ACTIVE|IN_PROGRESS|RISK_ACCEPTED|MITIGATING_CONTROL|FALSE_POSITIVE|CLOSED]`: Only list alerts in the specified states.  [default: OPEN, ACTIVE, IN_PROGRESS, RISK_ACCEPTED, MITIGATING_CONTROL, FALSE_POSITIVE]
+* `--state [OPEN|ACTIVE|IN_PROGRESS|RISK_ACCEPTED|MITIGATING_CONTROL|FALSE_POSITIVE|CLOSED]`: Only list alerts in the specified states.  [default: OPEN, IN_PROGRESS, RISK_ACCEPTED, MITIGATING_CONTROL, FALSE_POSITIVE]
```
Comment (edpin) [inline CLI.md:348]: And here, and all the ones below.

**Review by edpin — CHANGES_REQUESTED** [review]
I think something automatic got triggered and the ACTIVE state still shows up in the documentation, it's just not in the default anymore. Is that the desired end state?

**Comment by asieira** [comment]
> I think something automatic got triggered and the ACTIVE state still shows up in the documentation, it's just not in the default anymore. Is that the desired end state?

It is, since it's still listed as an accepted value in the [API documentation](https://api.zanshin.tenchisecurity.com/#tag/Alerts/operation/listAllAlert), so the SDK and CLI should still allow it.

**Review by edpin — APPROVED** [review]

**Review by lschanner-tenchi — APPROVED** [review]

**Pull request merged**